packages/web/src/pages/settings-page/components/desktop/ManagerMode/AccountsYouManageHomePage.tsx

```diff
@@ -0,0 +1,126 @@
+import { useCallback, useContext, useEffect } from 'react'
+
+import {
+  useApproveManagedAccount,
+  useGetManagedAccounts,
+  useRemoveManager
+} from '@audius/common/api'
+import { Status, UserMetadata } from '@audius/common/models'
+import { accountSelectors } from '@audius/common/store'
+import { Box, Divider, Flex, Text, TextLink } from '@audius/harmony'
+
+import LoadingSpinner from 'components/loading-spinner/LoadingSpinner'
+import { ToastContext } from 'components/toast/ToastContext'
+import { useSelector } from 'utils/reducer'
+
+import { AccountListItem } from './AccountListItem'
+import { sharedMessages } from './sharedMessages'
+import { AccountsYouManagePageProps, AccountsYouManagePages } from './types'
+const { getUserId } = accountSelectors
+
+const messages = {
+  takeControl:
+    'Take control of your managed accounts by making changes to their profiles, preferences, and content.',
+  noAccounts: 'You don’t manage any accounts.',
+  somethingWentWrong: 'Something went wrong. Please try again later.'
+}
+
+export const AccountsYouManageHomePage = ({
+  setPage
+}: AccountsYouManagePageProps) => {
+  const userId = useSelector(getUserId) as number
```

Comment:
* Is the cast because this might be undefined sometimes and we're trying to avoid a type error below?
* Should this be `as ID`? I know that just aliases to `number`, but seems to be the prevailing pattern everywhere and makes it easier to update the type of ID later on if we start using hashes everywhere.

Reply:
agreed with `as ID`

Reply:
* Yes - but I'm actually going to get rid of this here and move the cast to the line below for correctness, then have the query be disabled if user id is null or undefined
* Will change to ID!